contrib/format-hdf5/src/test/java/org/apache/drill/exec/store/hdf5/TestHDF5Format.java

```diff
@@ -98,9 +98,9 @@ public void testStarQuery() throws Exception {
 
     testBuilder()
       .sqlQuery("SELECT * FROM dfs.`hdf5/dset.h5`")
-      .unOrdered()
-      .baselineColumns("path", "data_type", "file_name", "int_data")
-      .baselineValues("/dset", "DATASET", "dset.h5", finalList)
+      .ordered()
```

Comment:
Better use unordered when you don't have order by clause in the query.

Reply:
Fixed here and elsewhere.